tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -222,9 +223,9 @@ func getIstiodLogs(client kube.ExtendedClient, config *config.BugReportConfig, r
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
-		clog, _, _, err := getLog(client, resources, config, namespace, pod, "discovery")
+		clog, _, _, err := getLog(client, resources, config, namespace, pod, common.DiscoveryContainerName)
 		appendGlobalErr(err)
-		writeFile(archive.IstiodPath(tempDir, namespace, pod+".log"), clog)
+		writeFile(filepath.Join(archive.IstiodPath(tempDir, namespace, pod), "discovery.log"), clog)
```

Comment:
what if I have multiple istiod? won't they all write to the same log?

Reply:
no, because all of the logs are under a dir which is the pod name